validator/accounts/v2/wallet_create.go

```diff
@@ -15,53 +15,55 @@ import (
 // CreateWallet from user input with a desired keymanager. If a
 // wallet already exists in the path, it suggests the user alternatives
 // such as how to edit their existing wallet configuration.
-func CreateWallet(cliCtx *cli.Context) error {
+func CreateWallet(cliCtx *cli.Context) (*Wallet, error) {
```

Comment:
Where is this returned wallet used?

Reply:
It's used as part of account create or import